### ISSUE TITLE
Integrate to JIRA database/view

### ISSUE THREAD

**Issue by usovdm-wiley** [vnabatov/pipl#6]
Nees to think...
It is an inspired point to integrate app and JIRA database (stories and task, creation also) by API.
And built app into the JIRA interface.

**Comment by vnabatov** [comment]
there is a way to load stories by the filter with `npm run load-jira`
another suggestion was to load tasks too (for example if another team created tasks and don't use the pipl)

**Comment by vnabatov** [comment]
current intergration is enough for now

**Comment by usovdm-wiley** [comment]
I agree. We should think about PI as an encapsulated event and import tasks once at the start.
